src/System.Net.Security/src/System/Net/SslStreamPal.Unix.cs

```diff
@@ -76,11 +76,21 @@ public static void QueryContextStreamSizes(SafeDeleteContext securityContext, ou
             streamSizes = new StreamSizes(Interop.libssl.SslSizes.HEADER_SIZE, Interop.libssl.SslSizes.TRAILER_SIZE, Interop.libssl.SslSizes.SSL3_RT_MAX_PLAIN_LENGTH);
         }
 
-        public static void QueryContextConnectionInfo(SafeDeleteContext securityContext, out SslConnectionInfo connectionInfo)
+        public static int QueryContextConnectionInfo(SafeDeleteContext securityContext, out SslConnectionInfo connectionInfo)
```

Comment:
@shrutigarg @vijaykota @bartonjs The PAL signature cannot change except if the common code is changed as well. 
The assumption that the return code is actually inspected is wrong: the Windows signature is still void and the error is actually ignored which might be a sign of a security issue.